examples/agent_client/examples/agent_client.rs

```diff
@@ -150,6 +150,12 @@ async fn make_agent() -> impl TalosAgent {
 async fn main() -> Result<(), String> {
     env_logger::builder().format_timestamp_millis().init();
 
+    log::info!("started program: {}", std::process::id());
+
+    info!("sleeping for 10 sec");
+    tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
Should this sleep code be merged into `master` ?

Reply:
Explained why in the log message.